docs/source/configuration/health-checks.mdx

```diff
@@ -5,94 +5,59 @@ description: Determining the router's status
 
 Health checks are often used by load balancers to determine whether a server is available and ready to start serving traffic.
 
-Because GraphQL queries can be sent as `GET` requests, a GraphQL query acts as an excellent health check. We recommend sending a simple query as a `GET` request with an appropriate `Content-type: application/json` header:
-
-```
-/?query={__typename}
+The Apollo Router supports a basic HTTP-level health check. This is enabled by default and is served on port `8088` at the URL path `/health`. This returns a `200` status code if the HTTP server is successfully serving.
+You can change this path by setting `health-check`:
```

Comment:
NIT: It's not a path. You could just leave out the word path to avoid worrying about the correct word:
```suggestion
You can change this by setting `health-check`:
```

Reply:
[aabb92a](https://github.com/apollographql/router/pull/1859/commits/aabb92a5d6e766149bec7286ab9cc6b64463c5ad)